File: manager-web/src/main/resources/static/js/service/typeTemplateService.js
// service层
// 创建一个service，名字叫brandService
app.service('typeTemplateService',function($http){
    // 增加
    this.add = entity=>{
        return $http.post('/typeTemplate/add',entity)
    }
    // 修改
    this.update = entity =>{
        return $http.post('/typeTemplate/modify',entity)
    }
    // 删除
    this.delete = ids => {
        return $http.post('/typeTemplate/delete',ids)
    }
    // 根据ID查询
    this.findById = id => {
        return $http.get(`/typeTemplate/${id}`)
    }
    // 分页查询所有
    this.findAll = (page,size,searchEntity)=>{
        return $http.post(`/typeTemplate/list?page=${page}&size=${size}`,searchEntity)
    }
    this.findOptionList = ()=>{
        return $http.get("/typeTemplate/list")
    }
})

File: sellergoods-service/src/main/java/com/zephon/sellergoods/service/TypeTemplateService.java
package com.zephon.sellergoods.service;

import com.github.pagehelper.PageInfo;
import com.zephon.pojo.TbSpecification;
import com.zephon.pojo.TbTypeTemplate;

import java.util.List;

/**
 * @author Zephon
 * @version V1.0
 * @Package com.zephon.sellergoods.service
 * @date 2020/6/30 上午10:44
 * @Copyright ©
 */
public interface TypeTemplateService {
    PageInfo<TbTypeTemplate> findAll(Integer page, Integer size, TbTypeTemplate typeTemplate);

    int add(TbTypeTemplate typeTemplate);

    TbTypeTemplate findById(Long id);

    int updateById(TbTypeTemplate typeTemplate);

    int deleteByIds(List<Long> ids);

    List<TbTypeTemplate> findAll();
}
